From: Hilko Bengen <bengen@debian.org>
Date: Fri, 17 May 2013 22:28:18 +0200
Subject: run: Turn error message about missing LIBGUESTFS_PATH into warning;
 output to STDERR

The previous behavior caused the build to abort when trying to build
the sysprep documentation -- without much of a hint of what went
wrong.
---
 run.in |    7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/run.in b/run.in
index 7545f0b..c4771b7 100755
--- a/run.in
+++ b/run.in
@@ -66,9 +66,10 @@ chcon --reference=/tmp tmp 2>/dev/null ||:
 if [ "x@ENABLE_APPLIANCE@" = "xyes" ]; then
     export LIBGUESTFS_PATH="$b/appliance"
 elif [ -z "$LIBGUESTFS_PATH" ]; then
-    echo "run: error: You used './configure --disable-appliance' so you must put an"
-    echo "run: error: appliance somewhere and set LIBGUESTFS_PATH to point to it."
-    exit 1
+    cat <<'EOF' >&2
+run: warning: You used './configure --disable-appliance' so LIBGUESTFS_PATH
+run: warning: has not been set automatically.
+EOF
 fi
 
 if [ -z "$LD_LIBRARY_PATH" ]; then
